api/src/main/java/io/minio/MinioClient.java

```diff
@@ -200,76 +208,80 @@
   private static final String US_EAST_1 = "us-east-1";
   private static final String UPLOAD_ID = "uploadId";
 
-  private static final Set<String> amzHeaders = new HashSet<>();
+  @SuppressWarnings("MismatchedQueryAndUpdateOfCollection")
```

Comment:
Why is it needed?

Reply:
It means that after initialization that collections don't used anywhere in code. Mb I just don't know about its use cases, so I just added a suppression.

Reply:
Can you send all clean ups and unrelated changes as separate PR?